definitions.yaml

```diff
@@ -182,6 +182,74 @@ definitions:
         type: string
         description: PEM-encoded certificate
 
+  # cluster metrics
+  V4GetClusterMetricsResponse:
+    type: object
+    properties:
+      container_count:
+        description: Number of containers running
+        $ref: '#/definitions/V4MetricsBundle'
+      pod_count:
+        description: Number of pods running
+        $ref: '#/definitions/V4MetricsBundle'
+      cpu_used:
+        description: CPU usage as a value between 0 and 1
```

Comment:
Later we have:
> Number of CPU cores used, as a floating point number

I don't follow how the number of cpu cores is between 0 and 1

Reply:
That is fixed now.